requirements.txt

```diff
@@ -2,5 +2,9 @@ mkdocs-material
 mkdocs>=1.5.0
 mkdocs-video
 mkdocs-include-markdown-plugin
-mkdocs-codeinclude-plugin>=0.2.1
-pymdown-extensions>=9.10
+mkdocs-codeinclude-plugin
```

Comment:
why did you drop the version requirements?

Reply:
my thought is generally:

```bash
python -m pip install SomePackage            # latest version
python -m pip install SomePackage==1.0.4     # specific version
python -m pip install 'SomePackage>=1.0.4'     # minimum version
```

from: [[User Guide - pip documentation v24.3.1](https://pip.pypa.io/en/stable/user_guide/)](https://pip.pypa.io/en/stable/user_guide/)

and I guess this will always try to get the most recent version so you don't have to keep it up to date

Reply:
- Table of Contents:
  ![table-of-contents](https://github.com/user-attachments/assets/408b5564-25dc-4b9e-bc7f-abfc025f6539)

The reason `Services` and `Machines` are bolder than the others is bc they contain links to their own `index.md` pages, e.g.

```yml
- Services:
    - Services: services/index.md
    - JupyterHub: services/jupyter-hub.md
    - Continuous Integration:
        - General: services/continuous-integration.md
        - Gitlab-CI: services/gitlab-ci.md

# ...

- Machines:
    - Machines: machines/index.md
    - Aurora/Sunspot:
```

Reply:
ahh yeah I had originally combined it into the navigation bar on the left, but then moved it back out in [70341e5](https://github.com/argonne-lcf/user-guides/pull/545/commits/70341e571ed628994989abecc5cdd11e7d146ca4)